shell_users/files/testuser_id_rsa.pub.template

```diff
@@ -0,0 +1 @@
+ssh-rsa <ssh key> test-key-id>[D[D[D[D[D[D[D[D[D[D[D[D<i[2~[Dtest-key-id>[D[D[D[D[D[D[D[Duser-key-id>[D[D[D[D[D[D[D[D[D[D[D[D[D[D[D[D[D[D[D[D[D[D[D
```

Comment:
Is this an encoding or conversion issue from Linux/Windows for the testuser_id_rsa.pub.template file? 
The following might have been closer to what you originally typed out?
```suggestion
ssh-rsa <ssh key><test-key-id></test-key-id><user-key-id>
```